blurp-app/src/pages/home.jsx

```diff
@@ -1,85 +1,96 @@
 import { Link } from 'react-router-dom';
-import BrandName from '../assets/blurp_noslogan.png'
-import NavbarProp from '../components/navbar'
-
+import BrandName from '../assets/blurp_noslogan_dark.png';
+import NavbarProp from '../components/navbar';
+import Spline from '@splinetool/react-spline';
+import SplineModel from '../assets/scene.splinecode';
 
 const Home = () => {
   return (
-    <>
-      <div class="background">
-        <NavbarProp/>
-        <div class="flex justify-center grid grid-cols-1 mx-24 px-6/12 py-10 w-5/12">
-          <div class="my-5">
-            <div class="grid justify-center">
-              <img class="m-4 h-40 w-56" src={BrandName} alt="Ecomapper the Greatest" />
-              {/* <h1 class="text-center main-title">Ecomapper</h1> */}
-              <h2 class="text-center main-subtitle">
+    <div className="landing-background">
+      <NavbarProp />
+      <div className="grid xl:grid-cols-2 grid-cols-1 justify-items-center">
+        <div className="relative grid justify-center grid-cols-1 mx-20 px-6/12 py-10 text-neutral-75 lg:w-5/6 order-last xl:order-first">
+          <div className="my-5">
+            <div className="grid justify-center">
+              <img class="m-4" src={BrandName} alt="Blurp the Greatest" />
```

Comment:
class --> to className here?

Reply:
I'm going to move forward with merge. Opening a new ticket to fix class->className across the code base.